app/presenters/content_item_presenter.rb

```diff
@@ -101,6 +101,10 @@ def service_manual_homepage?
     false
   end
 
+  def hide_from_search_engines?
```

Comment:
Is this definitely needed if the method is only going to be called from a publication-specific view?

Reply:
Oh bugger, no I didn't mean to commit that! Will remove sorry